File: spec/code_samples/Php_http1/alerts@{alert_type}/patch.php
<?php

HttpRequest::methodRegister('PATCH');
$request = new HttpRequest();
$request->setUrl('https://www.directfreight.com/api/alerts/%7Balert_type%7D');
$request->setMethod(HttpRequest::HTTP_METH_PATCH);

$request->setQueryData(array(
  'alert_id' => 'SOME_STRING_VALUE'
));

$request->setHeaders(array(
  'end-user-token' => 'SOME_STRING_VALUE'
));

$request->setBody('{"days_of_week":{},"end_hour":{},"expires_after":{},"list_order":{},"list_type":{},"minute_frequency":{},"send_type":{},"start_hour":{},"time_zone":{}}');

try {
  $response = $request->send();

  echo $response->getBody();
} catch (HttpException $ex) {
  echo $ex;
}

File: spec/code_samples/Python_requests/alerts@{alert_type}/patch.py
import requests

url = "https://www.directfreight.com/api/alerts/%7Balert_type%7D"

querystring = {"alert_id":"SOME_STRING_VALUE"}

payload = "{\"days_of_week\":{},\"end_hour\":{},\"expires_after\":{},\"list_order\":{},\"list_type\":{},\"minute_frequency\":{},\"send_type\":{},\"start_hour\":{},\"time_zone\":{}}"
headers = {'end-user-token': 'SOME_STRING_VALUE'}

response = requests.request("PATCH", url, data=payload, headers=headers, params=querystring)

print(response.text)